zerver/data_import/slack.py

```diff
@@ -1663,7 +1663,7 @@ def check_token_access(token: str, required_scopes: set[str]) -> None:
                 f"Slack token is missing the following required scopes: {sorted(missing_scopes)}"
             )
     else:
-        raise Exception("Unknown token type -- must start with xoxb- or xoxp-")
+        raise Exception("Enter a token. Valid tokens start with xoxb-.")
```

Comment:
"Invalid token" seems better than "Enter a token", since this error message is shared with folks starting on the command line.

Reply:
Okay, changed to use that.